README.md

```diff
@@ -51,7 +51,7 @@ System libraries:
     ```sh
     sudo dpkg --add-architecture i386
     sudo apt-get update
-    sudo apt-get install zlib1g-dev:i386
+    sudo apt-get install zlib1g-dev:i386 libssl-dev:i386
```

Comment:
The rest of the readme got borked

Reply:
honestly i have no clue what you want me to do with the readme (may just be due to sleep deprivation, tho), imma just revert it rn.

Reply:
just the `ldd` output, but i can do it myself